apps/percona/chaos/cstor_pool_scale/test.yml

```diff
@@ -85,7 +85,7 @@
           register: cvr_name1  
 
         - name: Scaling-down cstor-pool
-          shell: kubectl patch deployment {{ deployment_name.stdout }} -n {{ operator_ns }} --type=json -p='[{"op":"add", "path":"/spec/replicas", "value":"0"}]'
+          shell: kubectl scale deployment {{ deployment_name.stdout }} -n {{ operator_ns }} --replicas=0 
           register: result1
           failed_when: "'patched' not in result1.stdout"
```

Comment:
@shashank855 The check condition should be changed here as you are scaling 'patched' string wont be there in stdout right ?

Reply:
@dargasudarshan I made the change.